File: Lições/pascal/EXERCICIO03.pas
Program EXERCICIO03 ;

// 03) Escreva um algoritmo que receba uma temperatura em graus Celsius e converta-a para graus
// Farenheit. Exibir as duas temperaturas

Var
celsius, farenheit : real;

Begin
  WRITE('Digite a temperatura em Celsius: ');
  READ(celsius);
  farenheit := ((celsius * 9/5) + 32);
  WRITE('A temperatura ', celsius:0:2,'°C em farenheit fica : ', farenheit:0:2,'°');
End.

File: Lições/python/EXERCICIO03.py
# 03) Escreva um algoritmo que receba uma temperatura em graus Celsius e converta-a para graus 
# Farenheit. Exibir as duas temperaturas.

print('Celsius para Farenheit')
celsius = float(input('Digite a temperatura em °C: '))
farenheit = ((9/5 * celsius) + 32)
print('{}°C = {}°F'.format(celsius, farenheit))